src/pepr/policies/security.ts

```diff
@@ -16,6 +16,62 @@ import {
 import { exemptionAnnotationPrefix, isExempt, markExemption } from "./exemptions";
 
 const { containers } = sdk;
+
+/**
+ * This policy restricts the use of the Istio proxy user/group (1337) to only be used by Istio proxy containers.
+ * It allows specific Istio components (waypoints, ztunnel, and sidecars) to use these IDs.
+ * This prevents unauthorized pods from running with elevated privileges that could be used to bypass security controls.
+ */
+When(a.Pod)
+  .IsCreatedOrUpdated()
+  .Mutate(markExemption(Policy.RestrictIstioUser))
+  .Validate(request => {
+    if (isExempt(request, Policy.RestrictIstioUser)) {
+      return request.Approve();
+    }
```

Comment:
Not sure if we really want to handle this (and this comment probably applies to some of the other existing policies too):

but there are a number of other policies, that if you allow them, allow escalating to 1337. Like linux capabilities, privileged, or maybe running as root.

This PR probably isn't the place to discuss this, but honestly some of these almost belong in a hierarchy, or some dependencies between them